deploy.py

```diff
@@ -128,5 +160,26 @@ def create_control_node(config):
     # Upload all files in src/ to the bucket
     upload_cluster_scripts(config)
 
-    # Finally, deploy the control node cloudformation
-    create_control_node(config)
\ No newline at end of file
+    # Finally, deploy the control node CloudFormation
+    create_control_node(config)
+
+    upload_ssh_key(config['ssh_key_name'])
+
+    if not args.wait:
+        print("Deployment finished! Watch CloudFormation for details.")
+    else:
+        name = stack_name(config)
+        print("Waiting for your CloudFormation to finish contructing")
+        boto3.client('cloudformation').get_waiter('stack_create_complete').wait(
+            StackName=name)
+        outputs = boto3.resource('cloudformation').Stack(name).outputs
+
+        instance = next(
+            (x for x in outputs if x['OutputKey'] == 'Instance'), None)
+
+        if instance:
+            print("Connect to your control node: ssh -i {} ec2-user@{}".format(
+                config['ssh_key_name'],
+                boto3.resource('ec2').Instance(
+                    instance['OutputValue']).public_ip_address
+            ))
```

Comment:
I like this... maybe we should default `wait` to True?

Reply:
If this is the default behavior, that simplifies the SSH key secret handling. That's a good idea.

Reply:
> If this is the default behavior, that simplifies the SSH key secret handling.

I like that!